src/common/components/clusterConfiguration/DiscoveryImageConfigForm.tsx

```diff
@@ -48,7 +49,9 @@ export const StaticIPInfo: React.FC = () => {
   );
 };
 
-export type DiscoveryImageFormValues = ImageCreateParams & ProxyFieldsType;
+export type DiscoveryImageFormValues = ImageCreateParams &
```

Comment:
Please note there is a TS error in this file, when the constant `initialValues` is defined. 

I can understand it will be fixed after the API returns the certificate data in the infraEnv response.